cate/ops/data_frame.py

```diff
@@ -400,8 +417,28 @@ def great_circle_distance(p1: shapely.geometry.Point, p2: shapely.geometry.Point
     return math.atan2(y, x) / _DEG2RAD
 
 
-def _data_frame_geometry_op(instance_method, geometry: GeometryLike):
+# Transforms a geometry that is used in an operator for e.g. feature selection purposes. It assures
+# that both use an identical 'crs' (projection)
+def _transform_coordinates(geom, crs: dict):
+    if crs and 'init' not in crs:
+        raise ValidationError('No crs in GeoDataFrame' + str(crs))
+
+    if crs and crs['init'] != 'EPSG:4326':
```

Comment:
Gefahr. Ich denke es sollte heißen:

    if crs and 'init' in crs and str(crs['init']).lower() != 'epsg:4326':